gradio/layouts/tabs.py

```diff
@@ -76,6 +77,7 @@ def __init__(
             elem_id: An optional string that is assigned as the id of the <div> containing the contents of the Tab layout. The same string followed by "-button" is attached to the Tab button. Can be used for targeting CSS styles.
             elem_classes: An optional string or list of strings that are assigned as the class of this component in the HTML DOM. Can be used for targeting CSS styles.
             render: If False, this layout will not be rendered in the Blocks context. Should be used if the intention is to assign event listeners now but render the component later.
+            scale: relative size compared to adjacent elements. 1 or greater indicates the Tab will expand in size.
```

Comment:
I don't quite understand what `scale` means here. For a `Row`, I understand that its the scale relative to other rows in the same column. Is this understanding correct?
```suggestion
            scale: relative height compared to adjacent elements in the same column. 1 or greater indicates the Tab will expand in size to fill available screen height.
```

Reply:
If so, we can use similar language in `Row`

Reply:
The parent of a Row will almost certainly be a Column, so scale for a Row would configure the Row's height relative to other items in the Column. Doesn't really make sense to have Row's within a Row.
The parent of a Tab may be a Row or Column, so scale would affect the size of whichever way the layout is configured.

Reply:
Got it, I would just expand the docstrings to make this point clear